metron-platform/metron-common/src/main/java/org/apache/metron/common/bolt/ConfiguredParserBolt.java

```diff
@@ -36,4 +44,20 @@ protected SensorParserConfig getSensorParserConfig(String sensorType) {
     return getConfigurations().getSensorParserConfig(sensorType);
   }
 
+  @Override
+  public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
```

Comment:
Same as the comment for enrichment - do we want this tied to the bolts this way?